src/code/code_n64dd_800AD410.c

```diff
@@ -1 +1,19 @@
+#include "ultra64.h"
+#include "z64dma.h"
+#include "n64dd.h"
 
+void func_800ADA80(void) {
+    if (!B_80121AE0) {
+        DmaMgr_RequestSync(func_801C7740_unknown, (uintptr_t)D_00B8ADA0, D_00B9DA70 - D_00B8ADA0);
```

Comment:
This is _n64dd_SegmentStart/_n64dd_SegmentRomStart. Maybe we could include the n64dd baserom .o file in the spec for now?